src/Box/Box.cs

```diff
@@ -100,20 +102,18 @@ void OnSimulationEnded()
 		{
 			rigidBody.TopLevel = false;
 
-			rigidBody.Position = Vector3.Zero;
-			rigidBody.Rotation = Vector3.Zero;
-			rigidBody.Scale = Vector3.One;
+			rigidBody.Transform = Transform3D.Identity;
 
 			rigidBody.LinearVelocity = Vector3.Zero;
 			rigidBody.AngularVelocity = Vector3.Zero;
 
 			//Work around for #83 
 			if (enable_inital_pos)
```

Comment:
Rename the variable in this condition to `enable_inital_trasform`, so that it matches the action it encloses.